applications/GeoMechanicsApplication/custom_utilities/nodal_extrapolator.cpp

```diff
@@ -0,0 +1,111 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "nodal_extrapolator.h"
+#include "geometries/quadrilateral_2d_4.h"
+#include "geometries/triangle_2d_3.h"
+
+namespace Kratos
+{
+
+Matrix NodalExtrapolator::CalculateElementExtrapolationMatrix(GeometryType& rGeometry,
+                                                              GeometryData::IntegrationMethod IntegrationMethod) const
+{
+    CheckIfGeometryIsSupported(rGeometry);
+
+    auto          p_lower_order_geometry = CreateLowerOrderGeometry(rGeometry);
+    GeometryType& p_corner_geometry = p_lower_order_geometry ? *p_lower_order_geometry : rGeometry;
```

Comment:
Expected alignment on =, but apparently there is a rule that prevents the alignment.